src/tests/unit/tests/electron/platform/android/scan-results-helpers.ts

```diff
@@ -112,12 +112,14 @@ export function buildViewElement(
 
 export function buildRuleInformation(
     ruleId: string,
+    ruleLink = 'rule-link',
```

Comment:
Just curious, what is the difference in our case between the rule link and the guidance links? I thought this was guidance links. 😕

Reply:
My understanding is rule link is the place where the rule is defined and explained, and guidance links provide some extra infomation.